examples/BERT/utils.py

```diff
@@ -0,0 +1,58 @@
+import torch
+import torch.distributed as dist
+import os
+import torch.multiprocessing as mp
+import math
+
+
+def setup(rank, world_size, seed):
+    os.environ['MASTER_ADDR'] = 'localhost'
+    os.environ['MASTER_PORT'] = '12355'
```

Comment:
nit: this is specific to the particular distributed implementation

Reply:
It's set up to SLURM.